python/ray/_raylet.pyx

```diff
@@ -622,6 +614,13 @@ cdef class CoreWorker:
     def set_actor_title(self, title):
         self.core_worker.get().SetActorTitle(title)
 
+    def subscribe_to_plasma(self, plasma_event_handler):
+        self.plasma_event_handler = plasma_event_handler
+        self.core_worker.get().SubscribeToAsyncPlasma(async_plasma_callback)
+
+    def get_plasma_event_handler(self):
```

Comment:
do we need a getter here? can we just do `.plasma_event_handler`?

Reply:
I cannot access the member variable outside of the class, so I we need to keep the getter.